### PR TITLE
Displaying an alert popup everytime the Playlist is created or deleted

### PR DESCRIPTION
**Pull request by AmbrishRamachandiran**
## Hey, I just made a Pull Request!

Displaying an alert popup every time the Playlist is created or deleted, similar to other backstage activities.
The popup will be disappeared after some seconds
This will help users to experience good UI experience

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/playlist/src/components/CreatePlaylistButton/CreatePlaylistButton.tsx [hunk 1]
```diff
@@ -49,11 +55,16 @@ export const CreatePlaylistButton = () => {
       try {
         const playlistId = await playlistApi.createPlaylist(playlist);
         navigate(playlistRoute({ playlistId }));
+        alertApi.post({
+          message: `Added playlist '${playlist.name}'`,
+          severity: 'success',
+          display: 'transient',
+        });
```
Comment (kuangp) [inline plugins/playlist/src/components/CreatePlaylistButton/CreatePlaylistButton.tsx:62]: Can you also add this success message [here](https://github.com/backstage/backstage/blob/master/plugins/playlist/src/components/EntityPlaylistDialog/EntityPlaylistDialog.tsx#L134) as well 🙏 ? this is for when new playlists are created from the entity context

File: .changeset/purple-dolls-change.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-playlist': patch
+---
+
+Displaying an alert popup everytime the Playlist is created or deleted
```
Comment (kuangp) [inline .changeset/purple-dolls-change.md:5]:
```suggestion
Displaying an alert popup every time the Playlist is created or deleted
```

**Review by kuangp — CHANGES_REQUESTED** [review]
One additional request but this change is looking good so far! 😁

**Comment by AmbrishRamachandiran** [comment]
@kuangp changes pushed please review and let me know if any changes required

File: .changeset/purple-dolls-change.md
Comment (kuangp) [inline .changeset/purple-dolls-change.md:1]: Looks like 2 identical changesets were created? You should only need one and can delete the other

**Review by kuangp — CHANGES_REQUESTED** [review]
Looks like a failing test also needs to be addressed

**Comment by AmbrishRamachandiran** [comment]
@kuangp the latest commit have all the changes with all the test cases passed please review

**Comment by AmbrishRamachandiran** [comment]
@kuangp deleted duplicate files please review - TIA

**Review by kuangp — APPROVED** [review]
looks good! thanks for the contribution!

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Thanks both! :tada:

**Pull request merged**